var/spack/repos/builtin/packages/bash/package.py

```diff
@@ -35,6 +35,7 @@ class Bash(AutotoolsPackage):
         ('5.0', '011', '2c4de332b91eaf797abbbd6c79709690b5cbd48b12e8dfe748096dbd7bf474ea'),
     ]
 
+    # TODO: patches below are not managed by the GNUMirrorPackage base class
```

Comment:
I think at some point we need to consolidate per-version `urls`, `resources`, and `patch`es.  It seems like they should really be fetched the same way, or at least more logic should be shared so that it's easier to refactor for all three.

Reply:
I agree with that and I'd be happy to do some refactor later. We'll probably need to coordinate with @scheibelp (mirror everything) and @tldahlgren (parallel build) since at the moment refactoring this area will very likely affect both fetchers and stages (due to how much they're coupled together).